### ISSUE TITLE
wh branch does not converge when not loading it with COCO pretrained weight

### ISSUE THREAD

**Issue by alphadadajuju** [MCG-NJU/MOC-Detector#23]
Hi,

Firstly, thank you very much for your amazing work (and active responses in my earlier questions)!

I have a question regarding the "wh" branch that I'd like to consult with you. In my case, I noticed that this branch converged easily on JHMDB-21 when using COCO pretrained models (e.g., ResNet18). However, the total loss failed to converge if I did not load pretrained weights for the wh branch (while still loading COCO pretrained weights for the ResNet backbone).  

This phenomenon is not very intuitive for me. Mainly, the wh branch consists of relatively small number of parameters (~3x3x64x64 + 1x1x256x2K) when compared to the entire model. 

I wonder if you have experienced similar phenomenon? Is it normal that MOC's performance heavily depends on having used pretrained weights for the wh branch?

Thank you again.

**Comment by ArchiZX** [comment]
In our paper, the **ImageNet pretrain** results did not load pretrained weights for "wh" branch. And in my experiments the total loss can successfully converge. But I only try DLA-34 backbone rather than ResNet-18.

**Comment by alphadadajuju** [comment]
Thank you for sharing your experience! Here is something I observed (from little experiments I conducted on ResNet18). Perhaps it could be useful as a reference for others. 

When using COCO pretrained weights (including "wh" branch), the importance weight for the wh branch seems less relevant (0.1, 0.5, 1, etc). All these importance weights led to similar results for me on JHMDB split 1. 

However, if not loading COCO pretrained weight for wh branch, then the model converges better only when using the default importance weight 0.1 for the wh branch. When it was set to 1, my model failed to converge.  

Thanks again.
